docs/pages/User/GettingStarted/demo_k8s_on_desktop.md

```diff
@@ -1,32 +1,42 @@
 ---
-title: Demo with KubernetesOnDesktop
+title: Using LIQO to borrow resources
 weight: 4
 ---
 
-## Introduction
-This tutorial shows how to install [Liqo](https://liqo-io) on two [k3s](https://k3s.io/) clusters from scratch and then run a real application on the foreign cluster.
-In particular, we use a desktop application that has been _cloudified_ with the [KubernetesOnDesktop](#about-kubernetesondesktop) project, which aims at executing traditional Desktop applications in a remote environment, while keeping their graphical user interface locally.
+The usage of Liqo cna be adapted to more extended needs, as for example seamleassy share devices between laptops and PCs.
```

Comment:
I would cancel this sentence and move the content of the "scenario" section directly here.